### PR TITLE
Access options to request hooks (continues PR #115)

### PR DESCRIPTION
**Pull request by tasoskakour**
Continuing the PR to finish work in https://github.com/sindresorhus/ky/pull/115, by updating the tests.

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -49,7 +49,7 @@ export interface Options extends RequestInit {
 	Search parameters to include in the request URL.
 	Setting this will override all existing search parameters in the input URL.
 	*/
-	searchParams?: string | {[key: string]: string | number} | URLSearchParams;
+	searchParams?: string | { [key: string]: string | number } | URLSearchParams;
```
Comment (sindresorhus) [inline index.d.ts]: Don't do unrelated changes.

File: readme.md [hunk 2]
```diff
@@ -202,33 +202,29 @@ Hooks allow modifications during the request lifecycle. Hook functions may be as
...
 ###### hooks.afterResponse
 
 Type: `Function[]`<br>
 Default: `[]`
 
-This hook enables you to read and optionally modify the response. The hook function receives a clone of the response as the first argument. The return value of the hook function will be used by Ky as the response object if it's an instance of [`Response`](https://developer.mozilla.org/en-US/docs/Web/API/Response).
+This hook enables you to read and optionally modify the response. The hook function receives the request URL, the request options, and a clone of the response as the first argument. The return value of the hook function will be used by Ky as the response object if it's an instance of [`Response`](https://developer.mozilla.org/en-US/docs/Web/API/Response).
 
 ```js
-import ky from 'ky';
```
Comment (sindresorhus) [inline readme.md]: Don't do unrelated changes.

**Comment by sindresorhus** [comment]
> We still need to update how the options are destructured so all of the options actually end up on that options object passed to the hooks. - https://github.com/sindresorhus/ky/pull/115#issuecomment-480215940

**Comment by sindresorhus** [comment]
Ping @tasoskakour

**Comment by joaovieira** [comment]
Hadn't seen this effort before, but until the hook options are changed to include all the original options, I've updated the hooks options type in https://github.com/sindresorhus/ky/pull/161.

**Comment by sindresorhus** [comment]
Closing in favor of #163.
